sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -785,13 +785,36 @@ class CodegenContext {
    * @param expressions the codes to evaluate expressions.
    */
   def splitExpressions(row: String, expressions: Seq[String]): String = {
```

Comment:
To make it consistent, how about changing it to
```Scala
 def splitExpressions(row: String, arguments: Seq[(String, String)]): String = {
```

Reply:
I see. In addition to that, how about this since many caller passes `INPUT_ROW`?
```
def splitExpressions(row: String, arguments: Seq[(String, String)] = ("InternalRow", INPUT_ROW)): String = {
```
```